app/controllers/catalog_controller.rb

```diff
@@ -1,4 +1,8 @@
-# frozen_string_literal: true
+def add_facet_field_collection_sim?
+  return true if params["f"].blank?
+  return true if params["f"]["level_sim"].blank?
+  false
+end
```

Comment:
I think that is where I first defined it but it didn't work.  I searched inside the blacklight gem an found this example:

```
  ##
  # Determine whether to render the bookmarks control
  # (Needs to be available globally, as it is used in the navbar)
  def render_bookmarks_control?
    has_user_authentication_provider? && current_or_guest_user.present?
  end
```

I'll move inside the controller again to see if it will work.